src/app.ts

```diff
@@ -143,6 +185,28 @@ class FrameworkClient {
 			return command;
 		});
 		const imported = await Promise.all(commandsImports);
+
+		// Check to see if we are forced to reset all slash commands.
+		// This can be useful in the event a bad command got registered and needs to be removed.
+		if (this.options.slashCommandReset) {
+			await this.deleteSlashCommands();
```

Comment:
fetchBotCommands is called multiple times over the lifecycle of the bot, running this here will cause problems